index.html

```diff
@@ -360,40 +360,40 @@ <h6 class="description-text">I'm always interested in hearing about new projects
           </h6>
         </div>
         <div class="contact-form">
-          <form action="https://for,spree.io/f/{form_id}" id="form" method="post">
+          <form action="https://formsubmit.co/m.ziabakhteyari12@gmail.com" id="form" method="post">
             <input type="text" maxlength="30" name="fullname" id="fullname" placeholder="Full name" required>
             <input type="email" name="email" id="email" placeholder="Email address" required>
-            <textarea name="textarea" maxlength="500" id="textarea" cols="30" rows="10">Enter text here</textarea>
+            <textarea name="textarea" maxlength="500" id="textarea-text" cols="30" rows="10" placeholder="Enter text here"></textarea>
```

Comment:
Nice job implementing the placeholder! As requested by the previous reviewer, please add the `required` property to your `textarea` element. 😄

Reply:
This is my last chance to get approved, please check properly.

Reply:
Hello @Zia123456, the previous reviewer asked you to add the `required` property to your `textarea` element: 
![image](https://user-images.githubusercontent.com/42799579/193398242-04b0f3c9-7bb7-4460-ab25-5da4b046e047.png)

Reply:
But don't worry, re-reviews are meant to check if you added the implementations requested by the previous reviewer. Any additional request will be marked as [OPTIONAL] so you can focus on what the previous reviewer told you to do 😄. Taking this into account, if you do what the reviewer tells you, you are good to go 😉. Good luck on your journey!